aries_cloudagent/indy/models/cred_request.py

```diff
@@ -79,3 +81,178 @@ class Meta:
             "example": NUM_STR_WHOLE_EXAMPLE,
         },
     )
+
```

Comment:
Same comment again (re moving code to `formats/vc_di`).

Plus it looks like there is a bunch of code duplication (e.g. `AnoncredsLinkSecret` is defined again here)